egs2/talromur/tts1/run.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -euxo pipefail
+
+fs=22050
+n_fft=1024
+n_shift=256
+
+opts=
+if [ "${fs}" -eq 22050 ]; then
+    # To suppress recreation, specify wav format
+    opts="--audio_format wav "
+else
+    opts="--audio_format flac "
+fi
+
+
+train_config=conf/train.yaml
+inference_config=conf/decode.yaml
+
+g2p=g2p_is 
+
+./tts.sh \
+    --lang en \
+    --feats_type raw \
+    --fs "${fs}" \
+    --n_fft "${n_fft}" \
+    --n_shift "${n_shift}" \
+    --token_type phn \
+    --cleaner tacotron \
+    --g2p "${g2p}" \
+    --train_config "${train_config}" \
+    --inference_config "${inference_config}" \
+    ${opts} "$@"
+
+## Need to supply the following values in call to run.sh
+    # --train_set "${train_set}" \
+    # --valid_set "${valid_set}" \
+    # --test_sets "${test_sets}" \
+    # --expdir "${expdir}" \
+    # --srctexts "data/${train_set}/text" \
```

Comment:
I like setting the default values in order to allow running without modification.
Why don't you setting default speaker?

Reply:
I these values are supplied in the train_fastspeech2.sh and train_tacotron2.sh scripts depending on which model is being trained. The scripts do not need to be modified for this to work.
I could set the default model to be Tacotron but due to the usage of g2p transcription within the recipe you'd probably be better off running the train_tacotron2.sh script anyway.
I guess this kind of makes run.sh redundant because all it does is call tts.sh with some default parameters but I prefer having the model training scripts explicitly present in the directory.
I could have run.sh be a copy or symlink of train_tacotron2.sh and modify both training scripts to have them call tts.sh directly if that's preferrable?

Reply:
OK. I understand your motivation.
Then, could you update README to make users to know how to run this recipe?